tern/classes/image_layer.py

```diff
@@ -152,11 +162,42 @@ def remove_package(self, package_name):
             self.__packages.remove(self.__packages[rem_index])
         return success
 
+    def add_file(self, filedata):
+        if isinstance(filedata, FileData):
+            if filedata.path not in self.get_file_paths():
+                self.__files.append(filedata)
+        else:
+            raise TypeError('Object type is {0}, should be FileData'.format(
+                type(filedata)))
+
+    def get_file_paths(self):
+        '''Get the list of package names in this layer'''
```

Comment:
The documentation string is referring to packages instead of file paths

Reply:
Good catch. Thanks!